scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +657,105 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def __getitem__(self, val):
```

Comment:
I guess val->index or ind.

Reply:
This is just to emphasize that we can have an index, a slice, or an index array.